include/exadg/incompressible_navier_stokes/postprocessor/postprocessor.cpp

```diff
@@ -329,16 +397,16 @@ PostProcessor<dim, Number>::compute_mean_velocity(VectorType &       mean_veloci
 
 template<int dim, typename Number>
 void
-PostProcessor<dim, Number>::reinit_additional_fields(VectorType const & velocity)
+PostProcessor<dim, Number>::invalidate_derived_fields()
 {
-  vorticity.reinit(velocity);
-  divergence.reinit(velocity);
-  velocity_magnitude.reinit(velocity);
-  vorticity_magnitude.reinit(velocity);
-  streamfunction.reinit(velocity);
-  q_criterion.reinit(velocity);
-  cfl_vector.reinit(velocity);
-  mean_velocity.reinit(velocity);
+  vorticity.invalidate();
+  divergence.invalidate();
```

Comment:
I like that the ```reinit()``` function gets independent of velocity. I still think it is possible to use ```evaluate_get()``` without passing the vector that is needed for evaluation (see the comment above).